sqlglot/dialects/spark.py

```diff
@@ -101,6 +102,13 @@ class Tokenizer(Spark2.Tokenizer):
             for prefix in ("r", "R")
         ]
 
+        KEYWORDS = {
+            **Spark2.Tokenizer.KEYWORDS,
+            # as per https://spark.apache.org/docs/latest/sql-ref-syntax-qry-select-setops.html
+            # MINUS is an alias for EXCEPT
+            "MINUS": TokenType.EXCEPT,
+        }
```

Comment:
I see that `MINUS` is a synonym for `EXCEPT` on other dialects as well such as Oracle, Snowflake and Redshift; With this addition, I'd be inclined to revert all these overrides and generalize it by moving this mapping to `tokens.py` under `"EXCEPT"`.

Reply:
We can try this refactor, just make sure that `MINUS` can still be used as an identifier & projection alias in dialects that don't treat is as `EXCEPT`.